node/cn/tracers/internal/tracers/bigram_tracer.js

```diff
@@ -0,0 +1,62 @@
+// Modifications Copyright 2020 The klaytn Authors
```

Comment:
I think the license should be updated when we modified the file. If the file is exactly the same as the one in ethereum, we didn't put the modification comments.

Reply:
You're right.. I added klaytn copyright only to the modified files https://github.com/klaytn/klaytn/pull/752/commits/8a8c113dfca481a39163e4dc858039b9f42da3f4, Thanks!